Makefile.core.mk

```diff
@@ -425,6 +432,9 @@ operator-chart:
 	sed -i -e "s|^\(image: \).*$$|\1${IMAGE}|g" \
 	       -e "s/^\(  version: \).*$$/\1${VERSION}/g" chart/values.yaml
 
+github-workflow:
+	sed -i -e '1,/default:/ s/^\(.*default:\).*$$/\1 ${DEFAULT_CHANNEL}/' .github/workflows/release.yaml
```

Comment:
IIUC the `default` field in the workflow actually means a default value for channels which can be list of channels and not a DEFAULT_CHANNEL

Reply:
fixed! thank you